modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -479,4 +518,115 @@ export default class NestedTabs extends Base {
 
 		return !! $tabTitleContainerElement && isTabTitleActive ? true : false;
 	}
+
+	changeScrollStatus( event ) {
+		const slider = this.elements.$headingContainer[ 0 ];
+
+		if ( 'mousedown' === event.type ) {
+			slider.classList.add( 'e-scroll' );
+			slider.dataset.pageX = event.pageX;
+		} else {
+			slider.classList.remove( 'e-scroll' );
+			slider.dataset.pageX = '';
+		}
+	}
+
+	isHorizontalScroll() {
+		const slider = this.elements.$headingContainer[ 0 ];
+
+		return slider.clientWidth < this.getChildrenWidth( slider.children ) && 'enable' === this.getHorizontalScrollSetting();
```

Comment:
Would is be possible to use `slider.scrollWidth` instead of `this.getChildrenWidth()`? `slider.scrollWidth` will give you the full width of all scrollable elements together.